migration/files/00Initial_test.go

```diff
@@ -2,36 +2,22 @@ package files
 
 import (
 	"fmt"
-	"os"
-	"path/filepath"
 	"testing"
 
-	"github.com/centrifuge/go-centrifuge/utils"
+	"github.com/centrifuge/go-centrifuge/migration/utils"
```

Comment:
why a new package name, can we keep them as part of migrations package if no other package is using these ?

Reply:
I wanted to have a folder division for the migration files vs the migration wrapper. I am less interested on the package naming though, I am open to move it to migration or migrationfiles.